packages/app/src/components/inspector/PatternInspector.tsx

```diff
@@ -38,6 +39,7 @@ const Container = styled.div<{
   backgroundSize,
   imageRendering: 'crisp-edges',
   width: '100%',
+  transform: 'scale(0.999)',
```

Comment:
What does this do? I'm guessing it forces 3d rendering, but what does it do here?

Reply:
that was a test